src/pages/designing/kits/sketch.mdx

```diff
@@ -336,6 +345,154 @@ The first layer is the theme selection (i.e., 01 White theme). The second layer
+
+<Row className="resource-card-group">
+
+<Column colLg={4} colMd={4} noGutterSm>
+  <ResourceCard
+    subTitle="Gatsby theme Sketch kit"
+    href="sketch://add-library/cloud/304313c1-29a8-4946-a6f0-51dbec953bc2"
+    actionIcon="download">
+    <MdxIcon name="sketch" />
+  </ResourceCard>
+</Column>
+  
+<Column colLg={4} colMd={4} noGutterSm>
+  <ResourceCard
+    subTitle="Image production guideline Sketch kit"
+    href="https://www.sketch.com/s/240226b0-f455-484d-b412-cfd13fb1bb39"
+    actionIcon="launch">
+    <MdxIcon name="sketch" />
+  </ResourceCard>
+</Column>
+  
+</Row>
+
+### Native mobile
+
+These Carbon native mobile Sketch resources include everything you need to get
+your iOS or Android project underway. Install the grid template, be inspired by
+the patterns and examples, and get started with either the Light or Dark theme.
+
+<Row className="resource-card-group">
+
+<Column colLg={4} colMd={4} noGutterSm>
+  <ResourceCard
+    subTitle="Light theme for mobile"
+    href="sketch://add-library/cloud/s/a3343128-adb6-489c-9e62-709d89ba76e9"
+    actionIcon="download">
+    <MdxIcon name="sketch" />
+  </ResourceCard>
+</Column>
+<Column colLg={4} colMd={4} noGutterSm>
+  <ResourceCard
+    subTitle="Dark theme for mobile"
+    href="sketch://add-library/cloud/s/1f59f590-6915-47b0-bf06-6fd66209b3b3"
+    actionIcon="download">
+    <MdxIcon name="sketch" />
+  </ResourceCard>
+</Column>
+<Column colLg={4} colMd={4} noGutterSm>
+  <ResourceCard
+    subTitle="IBM Grid template for mobile"
+    href="https://www.sketch.com/s/42e694ee-4b37-41c9-8c8f-480e2415d9de"
+    actionIcon="launch">
+    <MdxIcon name="sketch" />
+  </ResourceCard>
+</Column>
+<Column colLg={4} colMd={4} noGutterSm>
+  <ResourceCard
+    subTitle="Mobile patterns and examples"
+    href="https://www.sketch.com/s/f13d67b9-3116-4b50-a2e9-56148f1976b0"
+    actionIcon="launch">
+    <MdxIcon name="sketch" />
+  </ResourceCard>
+</Column>
+</Row>
+
+### Accessibility
+
+The IBM Accessibility kit includes checklists, bite-sized guidance, and handoff
+assets to make sure your designs are accessible for implementation.
+
+<Row className="resource-card-group">
+
+<Column colLg={4} colMd={4} noGutterSm>
+  <ResourceCard
+    subTitle="IBM Accessibility Sketch kit"
+    href="https://www.sketch.com/s/f0a04c0d-fb62-4d71-92c6-07c402f8cae7"
+    actionIcon="launch">
+    <MdxIcon name="sketch" />
+  </ResourceCard>
+</Column>
+</Row>
+
+### Carbon Mid-Fi Sketch kit
+
+This kit provides designers with a way to visualize concepts and test them out
+before committing to high-fidelity designs.
+
+<Row className="resource-card-group">
+  <Column colMd={4} colLg={4} noGutterSm>
+    <ResourceCard
+      subTitle="Carbon Mid-Fi Sketch kit"
+      href="sketch://add-library/cloud/e888aa99-4d88-46a4-9b33-3a033e7f5ca2"
+      actionIcon="download"
+    >
+      <MdxIcon name="sketch" />
+    </ResourceCard>
+  </Column>
+</Row>
+
+### Text toolbar pattern library
```

Comment:
This, as a H3 heading feels not scalable. "Extensions" left more of a general bucket so these type of items. Also wondering if we even have this on here since it's already included on the [pattern page](https://carbondesignsystem.com/patterns/text-toolbar-pattern/#resources) instead.

Reply:
I agree. I'm going to delete this section.